src/operator/nn/mkldnn/mkldnn_base.cc

```diff
@@ -570,6 +570,32 @@ bool MKLDNNStorageType(const nnvm::NodeAttrs &attrs,
   return dispatched;
 }
 
+inline static const std::vector<NDArray> GetMKLDNNInputArray(const std::vector<NDArray> &inputs) {
+  std::vector<NDArray> ret;
+  ret.reserve(inputs.size());
+  for (const auto &in : inputs) {
+    if (in.IsView() && in.IsMKLDNNData()) {
+      ret.push_back(in.Reorder2Default());
+    } else {
+      ret.push_back(in);
+    }
+  }
+  return ret;
+}
+void MKLDNNRun(mxnet::FComputeEx fn,
```

Comment:
better to have a blank line before this function :)

Reply:
@rongzha1

Reply:
done